lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1304,7 +1341,9 @@ class MongoClientInterface {
             isVersioned: !!bucketInfo.getVersioningConfiguration(),
             ownerCanonicalId: bucketInfo.getOwner(),
         };
-
+        const isTransient =
+            this.config.getLocationConstraint(retBucketInfo.location)
```

Comment:
just to be safe, `this.config` can be undefined (given how it is set in constructor), so I feel we should check and make sure it is a valid object we have access to here